src/tests/test_magic_cmd.py

```diff
@@ -319,16 +350,67 @@ def test_table_schema_profile(ip, tmp_empty):
             assert cell == str(expected[profile_metric][0])
 
 
-def test_table_profile_warnings_styles(ip, tmp_empty):
-    ip.run_cell(
+def test_table_schema_profile_dbapi(ip_with_dbapi, tmp_empty):
+    sqlite_conn_b = sqlite3.connect("b.db")
+    ip_with_dbapi.push({"sqlite_conn_b": sqlite_conn_b})
+
+    ip_with_dbapi.run_cell(
+        """%%sql sqlite_conn_b
+CREATE TABLE t (n FLOAT);
```

Comment:
provide more descriptive names to table names

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/785/files#diff-9188e75e824219dd191cdc62d6d19f2b15c4bd0cf564abe3646ea907c32cd61dR361)